Dockerfile

```diff
@@ -10,9 +10,9 @@ RUN go mod verify
 
 COPY ./ ./
 
-RUN go build -ldflags="-w -s" -o server ./cmd
+RUN go build -ldflags="-w -s" -o bin ./cmd
 
 FROM scratch
-COPY --from=builder app/server /usr/server
+COPY --from=builder app/bin /usr/local/bin
 
-ENTRYPOINT [ "/usr/server" ]
+ENTRYPOINT [ "/usr/local/bin" ]
```

Comment:
bin is typically still a directory (check on your machine)

so you would want `/usr/local/bin/server`